src/reducers/entities/channels.js

```diff
@@ -181,33 +179,47 @@ function myMembers(state = {}, action) {
     }
 }
 
-function members(state = {}, action) {
-    const nextState = {...state};
-
+function membersInChannel(state = {}, action) {
```

Comment:
note to self: Rename `members` to `membersInChannel` wherever this is used in RN